src/patches/resource-extractor/cfnStore.ts

```diff
@@ -115,35 +115,30 @@ export class CfnStore {
    * @param region the AWS region to target
    * @returns CloudFormation Stack object
    */
-  private describeStack(
-    stackName: string,
-    region: string
-  ): CloudFormation.Stack | undefined {
-    let stack: CloudFormation.Stack | undefined;
+  private describeStack(stackName: string, region: string): Stack | undefined {
+    let stack: Stack | undefined;
     try {
       const output = cp.spawnSync(
-        'node',
+        'aws',
         [
-          '-e',
-          `
-            const sdk = require('aws-sdk');
-            const cfn = new sdk.CloudFormation({
-              apiVersion: '2016-11-15',
-              region: '${region}'
-            });
-            cfn.describeStacks({StackName: '${stackName}'})
-                .promise()
-                .then((data) => {
-                  const j = JSON.stringify(data);
-                  console.log(j);
-                }
-            );
-          `,
+          'cloudformation',
+          'describe-stacks',
+          '--stack-name',
+          stackName,
+          '--region',
+          region,
+          '--output',
+          'json',
         ],
         { encoding: 'utf8' }
       );
+
+      if (output.status !== 0) {
+        return undefined;
+      }
+
       const response = JSON.parse(output.stdout);
-      const stacks: CloudFormation.Stack[] = response.Stacks;
+      const stacks: Stack[] = response.Stacks;
 
       stack = stacks[0];
     } catch {}
```

Comment:
consider an improved error message

Reply:
Leaving as-is for now, it is common for this to "fail" because the stack isn't deployed yet